doc/mgmt/Docker to Host communication.md

```diff
@@ -2,7 +2,7 @@
 Docker to Host Communications
 
 # High Level Design Document
-#### Rev 0.7
+#### Rev 0.8
```

Comment:
If the client application decide to cancel a running request, how to? This is practical since gNMI service is serving a gNMI client, and the connection is broken suddenly. If continue running the request in host service may be meaningless or harmful, gNMI service may ask DBust host service to stop the request processing.

Reply:
I don't think dbus has this capability.